OpenRA.Mods.Common/UpdateRules/Rules/DefineOwnerLostAction.cs

```diff
@@ -0,0 +1,40 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class DefineOwnerLostAction : UpdateRule
+	{
+		public override string Name { get { return "Define OwnerLostActor on player actors"; } }
+		public override string Description
+		{
+			get
+			{
+				return "Actors now need OwnerLostActor trait to choose what will happen tho them when their owner loses.";
```

Comment:
Suggested wording: `"A new OwnerLostAction trait has been introduced to control what happens to a\nplayer's actors when they are defeated. A warning is displayed notifying that\nthis trait must be added to actors."`